common/traits/wc_race_traits.txt

```diff
@@ -0,0 +1,1488 @@
+		}
+	}
+}
+creature_hozen = {
+	index = 2050
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_hozen_desc
+			}
+			desc = trait_creature_hozen_character_desc
+		}
+	}
+}
+creature_draenei = {
+	index = 2051
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_draenei_desc
+			}
+			desc = trait_creature_draenei_character_desc
+		}
+	}
+}
+creature_arakkoa_corrupted = {
+	index = 2052
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_arakkoa_corrupted_desc
+			}
+			desc = trait_creature_arakkoa_corrupted_character_desc
+		}
+	}
+}
+
+# Undeads
+creature_abomination = {
+	index = 2053
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_abomination_desc
+			}
+			desc = trait_creature_abomination_character_desc
+		}
+	}
+}
+creature_lich = {
+	index = 2054
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_lich_desc
+			}
+			desc = trait_creature_lich_character_desc
+		}
+	}
+}
+creature_wickerman = {	# Throsic ghoul counterpart
+	index = 2055
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_wickerman_desc
+			}
+			desc = trait_creature_wickerman_character_desc
+		}
+	}
+}
+creature_wicker_behemoth = {	# Throsic vampire counterpart
+	index = 2056
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_wicker_behemoth_desc
+			}
+			desc = trait_creature_wicker_behemoth_character_desc
+		}
+	}
+}
+creature_wicker_golem = {	# Throsic abomination counterpart
+	index = 2057
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_wicker_golem_desc
+			}
+			desc = trait_creature_wicker_golem_character_desc
+		}
+	}
+}
+
+# Demons
+creature_annihilan = {
+	index = 2058
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_annihilan_desc
+			}
+			desc = trait_creature_annihilan_character_desc
+		}
+	}
+}
+creature_nathrezim = {
+	index = 2059
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_nathrezim_desc
+			}
+			desc = trait_creature_nathrezim_character_desc
+		}
+	}
+}
+creature_eredruin = {
+	index = 2060
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					NOT = { exists = this }
+				}
+				desc = trait_creature_eredruin_desc
+			}
+			desc = trait_creature_eredruin_character_desc
+		}
+	}
+}
+creature_fel_orc = {
```

Comment:
Dark background of this icon is kinda merges with dark ck3 interface background

Reply:
Changed art to a lighter Fel Orc picture to cause less blending with the background 460a2eaa0acff8acc5f38ca4e6188a3a7a3948f2